tests/keras/layers/convolutional_test.py

```diff
@@ -442,9 +440,8 @@ def test_convolution_3d():
                                'kernel_size': 3,
                                'padding': padding,
                                'strides': strides},
-                       input_shape=(num_samples,
-                                    input_len_dim1, input_len_dim2, input_len_dim3,
-                                    stack_size))
```

Comment:
I am guessing this will break other backend tests.
You have changed the input_shape.

Reply:
I have tested it on different backends(Tensorflow, MXNet) and it passes the test.